build/azure-pipelines/build-product.yml

```diff
@@ -80,63 +80,46 @@ stages:
         inputs:
           buildType: 'current'
           targetPath: '$(Pipeline.Workspace)'
-      - task: UseDotNet@2
-        inputs:
-          packageType: 'sdk'
-          version: '2.1.x'      
-      - task: EsrpCodeSigning@1
-        displayName: Sign Windows binary
+      - task: EsrpCodeSigning@2
         inputs:
           ConnectedServiceName: 'Code Signing'
           FolderPath: '$(Pipeline.Workspace)'
           Pattern: 'sqlcmd.exe'
           signConfigType: 'inlineSignParams'
-          SessionTimeout: '600'
-          MaxConcurrency: '5'
-          MaxRetryAttempts: '5'
           inlineOperation: |
             [
               {
-                "keyCode": "CP-230012",
+                "keyCode": "CP-230072",
```

Comment:
this is a test cert code, not a production cert code

Reply:
Updated to use production keycode.